ballerina/Package.md

```diff
@@ -1,14 +1,451 @@
 ## Package Overview
 
-The `jdbc` library is one of the external library packages of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
+This module provides the functionality that is required to access and manipulate data stored in any type of relational database,
```

Comment:
```suggestion
This package provides the functionality that is required to access and manipulate data stored in any type of relational database,
```
Check in other places as well

Reply:
Fixed in [9deb1b5](https://github.com/ballerina-platform/module-ballerinax-java.jdbc/pull/347/commits/9deb1b5ac0037560497ae32a399ccebaa2f2d8c2)